### PR TITLE
Scheduler: fix code style issues for pkg/scheduler

### PR DESCRIPTION
**Pull request by SataQiu**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
scheduler: fix code style issues for `pkg/scheduler`:
- uniform the receiver name as `sched`
- make the comments consistent with the function name

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/runtime/framework.go
Comment (AxeZhan) [inline pkg/scheduler/framework/runtime/framework.go:605]:
```suggestion
// PreEnqueuePlugins returns the registered preEnqueue plugins.
```

**Comment by sanposhiho** [comment]
Sorry about saying something troublesome, but please `/assign` an approver after you've got `/lgtm` from someone in a basic case. We want to make each PR go through two reviews basically. I'll check this PR after `/lgtm`-ed.

File: pkg/scheduler/framework/runtime/framework.go
Comment (SataQiu) [reply pkg/scheduler/framework/runtime/framework.go]: Thanks @AxeZhan Done!

**Comment by AxeZhan** [comment]
well, other things lgtm.
/lgtm

File: pkg/scheduler/apis/config/validation/validation_pluginargs.go
Comment (sanposhiho) [inline pkg/scheduler/apis/config/validation/validation_pluginargs.go:274]:
```suggestion
// ValidateVolumeBindingArgsWithOptions validates that VolumeBindingArgs and VolumeBindingArgsValidationOptions with scheduler features.
```

File: pkg/scheduler/testing/wrappers.go
Comment (sanposhiho) [inline pkg/scheduler/testing/wrappers.go:938]:
```suggestion
// FromPodSchedulingContexts creates a PodSchedulingContext wrapper from an existing PodSchedulingContext.
```

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

Thanks

**Pull request merged**
